update oauth_dispatch ADRs to accepted

robrap opened the pull request:
The following oauth_dispatch ADRs are being
marked as accepted, because they were implemented
and are in use in Open edX.
- 0006-enforce-scopes-in-LMS-APIs.rst
- 0007-include-organizations-in-tokens.rst

Given that, scopes have still not been widely
adopted in Open edX, which would require additional
decisions and investment.

nasthagiri commented:
@robrap These 2 ADRs are "accepted" in the sense that they have been implemented.  However, I do not believe we have widely adopted OAuth Scopes on our APIs as of yet.  There was an initial consumer for whom we expedited this implementation - but since then that initial consumer has taken a backstep - and we haven't further promoted/propagated Scopes.

robrap commented:
Thanks @nasthagiri. Note that we did implement [third-party auth scopes](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/oauth_dispatch/docs/decisions/0012-scope-and-filter-for-third-party-auth.rst) using this.
FYI: It was @MichaelRoytman who originally asked about the status of this ADR.

robrap commented:
@nasthagiri: 
1. I added "Accepted" to the other ADR.
2. I updated the PR description and commit comment with a note like you had added in the PR.  Let me know if it works for you.
Thanks.
